### ISSUE TITLE
Specify Bootstrap v4

### ISSUE THREAD

**Issue by ryanb** [wallaby-rails/wallaby-rails#201]
Is Wallaby compatible with Bootstrap 5? I got this error after running `bundle update wallaby-view`.

```
SassC::SyntaxError - Error: Undefined variable: "$emphasized-link-hover-darken-percentage".
        on line 131:7 of ../../.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/wallaby-6.1.3/app/assets/stylesheets/wallaby/base/_mixins.scss, in mixin `text-emphasis-variant`
        from line 4:10 of ../../.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/wallaby-6.1.3/app/assets/stylesheets/wallaby/components/_text.scss
        from line 16:9 of ../../.rbenv/versions/2.7.5/lib/ruby/gems/2.7.0/gems/wallaby-6.1.3/app/assets/stylesheets/wallaby/base.scss
        from line 2:9 of app/assets/stylesheets/wallaby/application.scss
>>   @if $emphasized-link-hover-darken-percentage != 0 {
```

I assume this is because the `bootstrap` gem updated to v5 when I updated wallaby. You might want to constrain wallaby to  bootstrap v4.

cc @tian-im

**Comment by tian-im** [comment]
Sure thing!

**Comment by ryanb** [comment]
@tian-im any update on this one?

**Comment by tian-im** [comment]
@ryanb, Sorry for the late update. I've completely forgot this one :p

New version is release to lock bootstrap at `~> 4.6`.

Could you take a look and see if it works for you?

**Comment by ryanb** [comment]
@tian-im I think this is resolved, but I'm seeing #170 again. Commented there.
